MAUI/Circular-Charts/DataLabels.md

```diff
@@ -93,4 +93,150 @@ chart.Series.Add(series);
+![Applying series interior for data label in MAUI chart](DataLabel_images/maui_chart_datalabels_useseriespalette.png)
+
+## Formatting Label Context
+
+The content of the label can be customized using the [LabelContext]() property. Following are the two options that are supported now,
+
+* [Percentage]() - This will show the percentage value of corresponding data point Y value
+
+* [YValue]() - This will show the corresponding Y value.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCircularChart>
+    
+    <chart:PieSeries ItemsSource="{Binding Data}" 
+                     LabelContext="Percentage"
+                     ShowDataLabels="True"
+                     XBindingPath="Product" 
+                     YBindingPath="SalesRate"/>  
+    
+</chart:SfCircularChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCircularChart chart = new SfCircularChart();
+    . . .
+    PieSeries series = new PieSeries()
+    {
+        ItemsSource = new ViewModel().Data,
+        XBindingPath = "Product";
+        YBindingPath = "SalesRate";
+        ShowDataLabels = true,
+        LabelContext = LabelContext.Percentage
+    };
+
+    chart.Series.Add(series);
+    this.Content = chart;
+        
+{% endhighlight %}
+
+{% endtabs %}
+
+![DataLabel context in MAUI Chart](DataLabel_images/maui_chart_datalabel_context.png)
+
+## LabelTemplate
+
+The [SfCircularChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCircularChart.html) provides support to customize the appearance of the datalabel by using the [LabelTemplate]() property.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCircularChart>
+
+    <chart:SfCircularChart.Resources>
+        <DataTemplate x:Key="LabelTemplate1">
+            <HorizontalStackLayout Spacing="5">
+                <Label Text="{Binding Item.Product}"
+                TextColor="White" 
+                FontAttributes="Bold"
+                FontSize="13"      
+				HorizontalOptions="Center"
+				VerticalOptions="Center"/>
+                <Label Text=" : " 
+				TextColor="White"
+				FontAttributes="Bold"
+				FontSize="13"
+				HorizontalOptions="Center"
+				VerticalOptions="Center"/>
+                <Label Text="{Binding Item.SalesRate}"
+				TextColor="White"
+				FontAttributes="Bold"
+				FontSize="13"
+				HorizontalOptions="Center"
+				VerticalOptions="Center"/>
+            </HorizontalStackLayout>
+        </DataTemplate>
+    </chart:SfCircularChart.Resources>
+
+    <chart:PieSeries ItemsSource="{Binding Data}" 
+                     ShowDataLabels="True"
+                     LabelTemplate="{StaticResource LabelTemplate1}"
+                     XBindingPath="Product" 
+                     YBindingPath="SalesRate">
+    </chart:PieSeries>
+
+</chart:SfCircularChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCircularChart chart = new SfCircularChart();
+    . . .
+    PieSeries series = new PieSeries();
+    series.ItemsSource = viewModel.Data;
```

Comment:
@EzhilarasanElangovan31 
Similar to other code snippet, update as new viewModel().Data